CHANGELOG.rst

```diff
@@ -9,6 +9,11 @@ Added
 * Add make command to autogen JSON schema from the models of action, rule, etc. Add check
   to ensure update to the models require schema to be regenerated. (new feature)
 
+Changed
+~~~~~~~
+
+* Switched mongo 4.0 as the default version starting with all supported OS's in ST2 3.3.0 @punkrokk
```

Comment:
Would be nice to follow the same Changelog format and keep it consistent. Looking at the older records:
```suggestion
* Switch to MongoDB ``4.0`` as the default version starting with all supported OS's in st2 ``v3.3.0`` (improvement) #4972
Contributed by @punkrokk 
```